src/jit/lsra.cpp

```diff
@@ -6377,6 +6397,45 @@ regNumber LinearScan::assignCopyReg(RefPosition* refPosition)
     return allocatedReg;
 }
 
+#ifdef _TARGET_ARM_
+// This is the function to check if assignedInterval of regRec is nullptr or not regardless of lastLocation.
+// So it would be call isUnassigned() with Maxlocation value.
+bool LinearScan::isUnassigned(RegRecord* regRec, Interval* newInterval)
+{
+    return isUnassigned(regRec, MaxLocation, newInterval);
+}
+#endif
+
+// Check if the assignedInterval was unassigned with checking nullptr and NextReflocation.
+bool LinearScan::isUnassigned(RegRecord* regRec, LsraLocation lastLocation, Interval* newInterval)
```

Comment:
The `newInterval` parameter is used only by `_TARGET_ARM_` so it would be better for it to be a different overload.
Also, it is used only for its type, so it would be better for the parameter to be a `RegisterType` - I think that makes more logical sense as well.